.github/workflows/ft-sync.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  push:
+    branches:
+      - main
+      - release-fixes
+    paths:
+      - "**.ftd"
+jobs:
+  everything:
+    name: ft-sync
+    runs-on: macos-10.15
+    steps:
+      - uses: actions/checkout@v2
+      - name: Running ft-sync status
+#      - uses: actions/checkout@v1
+        run: |
+          ls -la
+          chmod +x ./ft-sync
```

Comment:
Move the rust binary to .github folder. Also append platform name to binary, `ft-sync-linux-x64` or something.